### ISSUE TITLE
In Technical Report

### ISSUE THREAD

**Issue by univa-JASON** [OpenBMB/MiniCPM-V#421]
How did you proceed with DPO learning?
Using CPMTrainer, or HF DPOTrainer? 
Does CPM Trainer support DPO finetuning?

**Comment by yiranyyu** [comment]
We write the training code based on the RLAIF-V [project](https://github.com/RLHF-V/RLAIF-V). The code implement a trainer for DPO by itself.

**Comment by univa-JASON** [comment]
thank you for your answer! i have 1 more question, Can I apply wsd scheduler in This repo's finetuning code?

**Comment by DarioPTWR** [comment]
Hello! By any chance, could you point me to the code you used for MiniCPM-V DPO learning? Thank you and greatly appreciate it.
